lib/lz4.c

```diff
@@ -188,17 +191,38 @@ static void LZ4_write16(void* memPtr, U16 value) { ((unalign*)memPtr)->u16 = val
 
 static U16 LZ4_read16(const void* memPtr)
 {
+#ifdef __TI_COMPILER_VERSION__
+    /* Suppess warning: #551-D variable "ctx" is used before its value is set*/
```

Comment:
This is just compilation warning. I use arm_15.12.2.LTS compiler. You can download this compiler (it is bundled inside Code Composer Studio) http://processors.wiki.ti.com/index.php/Download_CCS